app/audit/Utilities.scala

```diff
@@ -48,8 +48,8 @@ object Utilities {
     (docDetail.getDocType, docDetail.documentText) match {
     case (_, Some(documentText)) if (documentText.contains("Class 2 National Insurance")) => Some("Class 2 National Insurance")
     case(_, Some(documentDescription)) if (documentDescription.contains("Cancelled PAYE Self Assessment")) => Some("Cancelled PAYE Self Assessment (through your PAYE tax code)")
-    case (Poa1Charge, _) => if (latePaymentCharge) Some("Late payment interest for payment on account 1 of 2") else Some("Payment on account 1 of 2")
-    case (Poa2Charge,_) => if (latePaymentCharge) Some("Late payment interest for payment on account 2 of 2") else Some("Payment on account 2 of 2")
+    case (Poa1Charge, _) => if (latePaymentCharge) Some("Late payment interest for payment on account 1 of 2") else Some("First payment on account")
```

Comment:
Could you use messages here? For if we need to change them in future?

Reply:
Investigated, messages -> charge types doesn't match 1-to-1 so would need to define these strings locally anyway